ibm_catalog.json

```diff
@@ -14,27 +14,26 @@
       ],
       "keywords": [
         "vpc",
-        "slz",
         "IaC",
         "infrastructure as code",
         "terraform",
         "solution",
         "VPN",
         "OpenVPN"
       ],
-      "short_description": "Creates client-to-site VPN connectivity to landing zones using VPN Server",
-      "long_description": "By default, the landing zone patterns are configured with private networks not available over the internet. To access these networks, there are several connectivity options. This deployable architecture pattern configures the client-to-site VPN Server connectivity with only a few required inputs to configure it within an existing landing zone. Once deployed, you can install an OpenVPN client application and import a profile from the VPN Server on the devices you want to access the VPN.  The configuration can include a list of users that will be provided access to the private network, controlled by IBM Cloud IAM.",
+      "short_description": "Creates client-to-site VPN connectivity to VPC",
+      "long_description": "Some VPC patterns are configured with private networks not available over the internet. To access these networks, there are several connectivity options. This deployable architecture pattern configures the client-to-site VPN Server connectivity with only a few required inputs to configure it within an existing VPC. Once deployed, you can install an OpenVPN client application and import a profile from the VPN Server on the devices you want to access the VPN.  The configuration can include a list of users that will be provided access to the private network, controlled by IBM Cloud IAM.",
       "offering_docs_url": "https://cloud.ibm.com/docs/secure-infrastructure-vpc?topic=secure-infrastructure-vpc-connect-landingzone-client-vpn",
       "offering_icon_url": "https://broker-staging.vpnaasdev.cloud.ibm.com/C2S_icon.svg",
       "provider_name": "IBM",
       "features": [
         {
-          "description": "Supports creating and configuring a new Secrets Manager instance, or using an existing one, to create a secret group and a new private cert.",
-          "title": "Creates and configures an optional Secrets Manager instance, or uses existing."
+          "description": "Supports using an existing Secrets Manager instance, to create a secret group and a new private cert.",
+          "title": "Configures existing Secrets Manager instance, and create a secret group and a new private cert."
         },
         {
           "description": "The network ACL on this subnet grants all access from any source.",
-          "title": "A subnet named 'client-to-site-subnet' in the landing-zone management VPC"
+          "title": "A subnet named 'client-to-site-subnet' in the VPC"
```

Comment:
Is this subnet hardcoded?

Reply:
yes https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/pull/329/files#diff-3fd60e5e096ce4345846d75a09c0e503d86632aaa408f0acf606b74253728b9fR158